src/ApparentHorizons/SpherePackIterator.hpp

```diff
@@ -0,0 +1,106 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+///
+/// \file
+/// Defines class SpherePackIterator.
+
+//
+// iterate through an array with the layout of the Ylm-spectral
+// coefficients of SpherePack.
+//
+// Note: SpherePackIterator.cpp contains an explanation of the
+// SpherePack memory layout.
+//================================================================
```

Comment:
I would remove the above comment: 
* It is partially redundant with the Doxygen comment.
* Pointing to the `.cpp` for the SpherePack memory layout comment doesn't strike me as very useful... anyone needing to know those details will already be looking at the `.cpp`.